server/container/guice/guice-common/src/main/java/org/apache/james/utils/ExtendedServerProbe.java

```diff
@@ -40,4 +41,6 @@ ComposedMessageId appendMessage(String username, MailboxPath mailboxPath, InputS
 
     void addActiveSieveScript(String user, String name, String script) throws Exception;
 
+    Collection<String> listSub(String user) throws Exception;
```

Comment:
We don't call IMAP to get that list so "listSubscriptions" will be better then I changed